### DISCUSSION TITLE
SQLBuilder - How to Select DB Provider

### DISCUSSION THREAD

**Discussion by mateusbello** [maxtoroq/DbExtensions#9] [Q&A]
Hi,

When building a query, how do I set the db provider?
For example, queries are built differently on MSSQL and MYSQL, how do I set the provider so it can generate DB specific query?

Thanks

**Comment by maxtoroq** [comment]
SqlBuilder always generates the same SQL, it knows nothing about providers. Perhaps you are referring to SqlSet, which uses the database connection to detect providers and currently has no way to specify one explicitly.
